From 0808069b35550fac33e43393039d799fa78f84a5 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:12:14 -0400
Subject: [ethernet] igb: Cleanups to remove return parentheses

Message-id: <1409062359-13063-57-git-send-email-sassmann@redhat.com>
Patchwork-id: 89165
O-Subject: [RHEL7.1 PATCH 56/81] igb: Cleanups to remove return parentheses
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Carolyn Wyborny <carolyn.wyborny@intel.com>

This patch fixes ERROR:RETURN_PARENTHESES from checkpatch file check.

Signed-off-by: Carolyn Wyborny <carolyn.wyborny@intel.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit da1f1dfeb36550c5725ab62fcb8dca1c48ff34ba)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/igb_main.c b/drivers/net/ethernet/intel/igb/igb_main.c
index abec14f..3278c93 100644
--- a/drivers/net/ethernet/intel/igb/igb_main.c
+++ b/drivers/net/ethernet/intel/igb/igb_main.c
@@ -574,7 +574,7 @@ static int igb_get_i2c_data(void *data)
  struct e1000_hw *hw = &adapter->hw;
  s32 i2cctl = rd32(E1000_I2CPARAMS);
 
- return ((i2cctl & E1000_I2C_DATA_IN) != 0);
+ return !!(i2cctl & E1000_I2C_DATA_IN);
 }
 
 /**
@@ -638,7 +638,7 @@ static int igb_get_i2c_clk(void *data)
  struct e1000_hw *hw = &adapter->hw;
  s32 i2cctl = rd32(E1000_I2CPARAMS);
 
- return ((i2cctl & E1000_I2C_CLK_IN) != 0);
+ return !!(i2cctl & E1000_I2C_CLK_IN);
 }
 
 static const struct i2c_algo_bit_data igb_i2c_algo = {
@@ -7056,7 +7056,7 @@ static bool igb_clean_rx_irq(struct igb_q_vector *q_vector, const int budget)
  if (cleaned_count)
   igb_alloc_rx_buffers(rx_ring, cleaned_count);
 
- return (total_packets < budget);
+ return total_packets < budget;
 }
 
 static bool igb_alloc_mapped_page(struct igb_ring *rx_ring,
-- 
1.7.1